src/components/checkboxes/index.md.njk

```diff
@@ -32,6 +32,10 @@ If you are asking just [one question per page](../../patterns/question-pages/#st
 
 Read more about [why and how to set legends as headings](../../get-started/labels-legends-headings).
 
+If you are asking several questions on one page and the `<legend>` is not the page title, do not set the contents of the `<legend>` as the page heading:
```

Comment:
This is more a content question but it feels a bit weir to refer to a coded example here since further down the page we explain what the coded examples are: 
> There are 2 ways to use the checkboxes component. You can use HTML or, if you’re using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.